app/views/profile/friends/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Friends Profile</h1>
```

Comment:
Maybe this could have the friend's name instead of 'Friends profile' to be a little more pesonable